### PR TITLE
Add installed versions in "install" or "update" ComboBox version listing

### PR DESCRIPTION
**Pull request by rrelyea**
Fixes NuGet/Home#2124.
Fixes NuGet/Home#1740.

@rohit21agrawal @alpaix @yishaigalatzer

### CODE REVIEW

File: src/NuGet.Clients/PackageManagement.UI/VersionForDisplay.cs [hunk 1]
```diff
@@ -46,6 +50,11 @@ public class DisplayVersion
             }
         }
 
+        public bool IsCurrentInstalled
```
Comment (yishaigalatzer) [inline src/NuGet.Clients/PackageManagement.UI/VersionForDisplay.cs]: nit: keep on the same line

**Comment by yishaigalatzer** [comment]
Looks good, but I'm missing the UI behaviors when the current version is selected, is it just the install button being disabled?

Should we add a tooltip in that case so a user can figure out what it is grayed out?

File: src/NuGet.Clients/PackageManagement.UI/Converters/BooleanToFontWeightConverter.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
+﻿using System;
```
Comment (alpaix) [inline src/NuGet.Clients/PackageManagement.UI/Converters/BooleanToFontWeightConverter.cs]: add headers

**Comment by alpaix** [comment]
:shipit:

**Pull request merged**